src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -108,33 +108,32 @@ private unsafe Stream GetBitmapStream(ReadOnlySpan<byte> rawData)
         {
             try
             {
+                short signature = MemoryMarshal.Read<short>(rawData);
+
+                if (signature != 0x1c15 || rawData.Length < sizeof(OBJECTHEADER))
```

Comment:
yo'ure checking size here, but not checking that `rawData` is itself large enough to read a short (line 111 will throw)

Reply:
Ah I see you catch below. In that case you don't need tp check `rawData.Length < sizeof(OBJECTHEADER)` because on 119 it will throw ArgumentEXception which you will catch and return null anyway.